### PR TITLE
implemented bcrypt

### PR DESCRIPTION
**Pull request by SietzeKoonstra**

### CODE REVIEW

File: Server/DatabaseHandler.cs [hunk 1]
```diff
@@ -219,7 +219,7 @@ public static string GetPasswordFromClient(string email)
                     connection.Open();
                     return (string)command.ExecuteScalar();
                 }
-            } catch (Exception e)
+            } catch (Exception)
```
Comment (Rik2001) [inline Server/DatabaseHandler.cs:222]: waarom de e geremoved?

File: Server/Server.cs [hunk 2]
```diff
@@ -182,7 +182,8 @@ public int CreateAndAddClient(string userName, string email, string password)
                 Email = email,
                 Name = userName
             };
-
+            //hashedPassword = Helper.HashPassword(password);
+            //client.Id = DatabaseHandler.AddClient(client, hashedPassword);
```
Comment (Rik2001) [inline Server/Server.cs:186]: comments kunnen weg

**Review by Rik2001 — CHANGES_REQUESTED** [review]

File: Server/DatabaseHandler.cs (hunk 1 above)
Comment (SietzeKoonstra) [reply Server/DatabaseHandler.cs:222]: it's never used.

**Review by JochemFB — APPROVED** [review]

**Review by Rik2001 — APPROVED** [review]
lgtm

**Pull request merged**